### PR TITLE
Update readme on airbag after 1st test run

### PR DESCRIPTION
**Pull request by jamesgorrie**
Updated the readme after @gestchild took it for the first test run.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@gestchild How\s this look?

File: infra/README.md [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+      aws_access_key               = "KEY"
+      aws_secret_key               = "SECRETSHHH"
+      wellcomecollection_key_path  = "/Users/hank/.ssh/key.pub"
+      wellcomecollection_key_name  = "KEY_NAME"
 
-Be very careful of not checking these in, `tfvars` are excluded from git at the root level,
-but probably worth mentioning.
+* Aquire the public key from a friendly Wellcome Collection developer near you.
+  
+  Save it somewhere and If you noticed there is a reference to a key file there, you will need to get that too.
```
Comment (gestchild) [inline infra/README.md]: Not entirely sure what this line means, isn't there always a key file?

Comment (gestchild) [inline infra/README.md]: @jamesgorrie ^^

Comment (jamesgorrie) [inline infra/README.md]: Updated that - does the new sentence make sense?

Comment (jamesgorrie) [inline infra/README.md]: @gestchild thoughts?

Comment (gestchild) [inline infra/README.md]: @jamesgorrie looks good to me

**Pull request merged**